Explorer/Assets/Scripts/Global/Dynamic/IRealmController.cs

```diff
@@ -15,6 +15,8 @@ public interface IRealmController
         /// </summary>
         UniTask SetRealmAsync(URLDomain realm, Vector2Int playerStartPosition, AsyncLoadProcessReport loadReport, CancellationToken ct);
```

Comment:
Instead of Dynamic Scene just move the direct realm change to the debug menu panel with other cheats